src/Symfony/Component/Link/composer.json

```diff
@@ -0,0 +1,39 @@
+{
+    "name": "symfony/link",
+    "type": "library",
+    "description": "Symfony Link Component",
+    "keywords": ["link", "http", "HTTP/2", "preload", "prefetch", "pre-render", "dns-prefetch", "push", "performance"],
+    "homepage": "https://symfony.com",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Kévin Dunglas",
+            "email": "dunglas@gmail.com"
+        },
+        {
+            "name": "Symfony Community",
+            "homepage": "https://symfony.com/contributors"
+        }
+    ],
+    "require": {
+        "php": ">=5.5.9"
+    },
+    "suggest": {
+        "symfony/http-kernel": ""
+    },
+    "require-dev": {
+        "symfony/http-kernel": "^2.8 || ^3.0"
```

Comment:
The EventDispatcher and HttpFoundation components are used by the `LinkListenerTest` class too.